elastic/logs/challenges/logging-streams.json

```
@@ -0,0 +1,315 @@
+              }
+            },
+            {
+              "normalize_for_stream": {}
+            },
+            {
+              "dot_expander": {
+                "path": "resource.attributes",
+                "field": "*",
+                "ignore_failure": true
+              }
+            },
+            {
+              "dot_expander": {
+                "path": "attributes",
+                "field": "*",
+                "ignore_failure": true
+              }
+            },
+            {
+              "script": {
+                "source": "ctx[\"stream.name\"] = params.field",
+                "lang": "painless",
+                "params": {
+                  "field": "logs"
+                }
+              }
+            },
+            {
+              "pipeline": {
+                "name": "logs@stream.reroutes",
+                "ignore_missing_pipeline": true
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "logs-index-template",
+      "operation": {
+        "operation-type": "create-composable-template",
+        "template": "logs@stream",
+        "body": {
+          "index_patterns": [
+            "logs"
+          ],
+          "template": {
+            "settings": {
+              "index": {
+                "default_pipeline": "logs@stream.processing"
+              }
+            },
+            "mappings": {
+              "properties": {
+                "stream.name": {
+                  "type": "constant_keyword",
+                  "value": "logs"
+                }
+              }
+            }
+          },
+          "composed_of": [
+            "logs@stream.layer"
+          ],
+          "priority": 1000,
+          "version": 1,
+          "_meta": {
+            "managed": true,
+            "description": "The index template for logs stream"
+          },
+          "data_stream": {
+            "hidden": false,
+            "allow_custom_routing": false
+          }
+        }
+      }
+    },
+     {
+      "name": "setup",
+      "operation": {
+        "operation-type": "put-pipeline",
+        "id": "logs@custom",
+        "body": {
+          "description": "Sends all data to the 'logs' index. This is easier than changing the index name in the source.",
+          "processors": [
+            {
+              "reroute": {
+                "destination": "logs"
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "name": "bulk-index",
+      "operation": {
+        "operation-type": "raw-bulk",
+        "param-source": "processed-source",
+        "time-format": "milliseconds",
+        "profile": "fixed_interval",
+        "bulk-size": {{ p_bulk_size }},
+        "detailed-results": true
+      },
+      "clients": {{ p_bulk_indexing_clients }}{% if p_throttle_indexing %},
+      "ignore-response-error-level": "{{error_level | default('non-fatal')}}",
+      "schedule": "timestamp-throttler",
```

```suggestion
      "clients": {{ p_bulk_indexing_clients }},
      "ignore-response-error-level": "{{error_level | default('non-fatal')}}",
      {% if p_throttle_indexing %}
      "schedule": "timestamp-throttler",
```

I know this was copied over, but let's not repeat the mistake of combining throttling with ignore errors configuration. These are orthogonal things.